fds-lambda-ml-integration/src/main/java/com/provectus/fds/ml/JobDataGenerator.java

```diff
@@ -23,13 +22,9 @@ CsvRecordProcessor generateTrainingData(IntegrationModuleHelper h, boolean enabl
                 .withClientExecutionTimeout(Integer.parseInt(h.getConfig(CLIENT_EXECUTION_TIMEOUT, CLIENT_EXECUTION_TIMEOUT_DEF)));
 
         AmazonAthenaClientBuilder athenaClientBuilder = AmazonAthenaClientBuilder.standard()
-                .withRegion(h.getConfig(ATHENA_REGION_ID, ATHENA_REGION_ID_DEF))
+                .withRegion(System.getenv("ATHENA_REGION_ID"))
```

Comment:
keep configs in separate class

Reply:
@germanosin do you want a separete Config for each lamda or for the whole module?

I think, if we create a single class configuration it will lead to the hard supported configuration.

Reply:
fixed